docs/html/development/getting-started.rst

```diff
@@ -76,26 +76,6 @@ To use linters locally, run:
 .. code-block:: console
 
     $ tox -e lint
-    $ tox -e lint-py2
-
-The above commands run the linters on Python 3 followed by Python 2.
-
-.. note::
-
-    Do not silence errors from flake8 with ``# noqa`` comments or otherwise.
```

Comment:
Do we not still want this note?

Reply:
Hmm... We probably do.

There's a check in pre-commit-pygrep (a repo containing a bunch of hooks) that disallows "broad" `# noqa` comments. Maybe we can adopt that check? It's still not telling folks to not use noqa style comments though and we don't have any kind of data on how useful that comment is.

This is slightly tricky to fit into the current flow of the document, maybe we can just keep the comment after the linter? Maybe it's okay to drop this?

Uhh...

Reply:
Actually, let's figure this out in a follow up PR? I'd like to not block this PR on this (minor) item, but I definitely think we should add this back, in some form.

Reply:
I'm gonna take a bit of liberty here and assume you're okay with tackling this in a follow up given that there's a green tick on your review. :3

I'll file an issue to follow up on this. :)

Reply:
Sure, that works for me. As for wording, I'd leave the note where it is and word it as something like

> Avoid using ``# noqa`` comments to suppress linter warnings - wherever possible, warnings should be fixed instead, with ``# noqa`` reserved for rare cases where the recommended style causes severe readability problems.

(Yes, it's a bit wordier, but I think that adding a bit more detail to the comment makes it fit more naturally - you may disagree, I'm not going to insist 🙂).

Reply:
> you may disagree

Nah, I agree. I'll just use those words in a PR that I'll hopefully file in a few minutes.

Reply:
Filed: https://github.com/pypa/pip/issues/7411